source/python/setup.py

```diff
@@ -0,0 +1,31 @@
+"""
+One-time setup script to download classifier models and pre-populate database with data neccessary for fact extraction.
+"""
+
+from sqlalchemy import create_engine
+from idetect.model import db_url, Base, Session, Country, FactKeyword
+from idetect.load_data import load_countries, load_terms
+from idetect.nlp_models.category import CategoryModel, Tokenizer, TfidfTransformer, LsiTransformer
+from idetect.nlp_models.relevance import RelevanceModel, Stemmer, Combiner, RelevanceKeyWordClassifier
+
+if __name__ == "__main__":
+
+    # Create the Database
+    engine = create_engine(db_url())
+    Session.configure(bind=engine)
+    Base.metadata.create_all(engine)
+
+    session = Session()
+    # Load the Countries data if necessary
+    if not session.query(Country).first():
+        load_countries(session)
+
+    # Load the Keywords if neccessary
+    if not session.query(FactKeyword).first():
+        load_terms(session)
+
+    session.close()
```

Comment:
Is it necessary to explicitly close the session?

Reply:
No, since the program is about to exit anyway, but it's a good practice to do so.